### PR TITLE
CLN: reorganize ci_testing

### PR DESCRIPTION
**Pull request by seanpmorgan**

### CODE REVIEW

**Comment by facaiy** [comment]
It seems that both check_futures_test.py and file_name_test.py only are invoked by ci_sanity.sh. How about putting those files in the `ci_build`? Maybe we can create a subdirectory for them if necessary, say `ci_build/check`, `ci_build/verify` or something else.

And we can delete `_test` suffix which looks kind of misleading, what do you think, Sean?

**Comment by seanpmorgan** [comment]
> It seems that both check_futures_test.py and file_name_test.py only are invoked by ci_sanity.sh. How about putting those files in the `ci_build`? Maybe we can create a subdirectory for them if necessary, say `ci_build/check`, `ci_build/verify` or something else.
> 
> And we can delete `_test` suffix which looks kind of misleading, what do you think, Sean?

I like that. Updated.

File: tools/ci_build/ci_sanity.sh [hunk 1]
```diff
@@ -227,13 +227,13 @@ do_bazel_nobuild() {
...
-    python check_futures_test.py
+    cd "$ROOT_DIR/tools/ci_build/verify"
+    python check_futures.py
 }
 
 do_check_file_name_test() {
-    cd "$ROOT_DIR/tools/test"
-    python file_name_test.py
+    cd "$ROOT_DIR/tools/ci_build/verify"
+    python file_name.py
```
Comment (facaiy) [inline tools/ci_build/ci_sanity.sh]: nit: check_file_name.py ?

**Comment by WindQAQ** [comment]
Got  the error `AttributeError: 'module' object has no attribute 'keras' ` for CI tests. Probably an issue from upstream?

**Pull request merged**